packages/api/src/controllers/stripe.ts

```diff
@@ -120,7 +121,7 @@ async function reportUsageForUser(
   });
 
   if (
-    user.stripeProductId.includes("hacker_1") ||
+    (actuallyReport && user.stripeProductId.includes("hacker_1")) ||
```

Comment:
Shouldn't `actuallyReport` be respected when the product ID is `prod_O9XuIjn7EqYRVW` as well? Maybe a parentheses issue going on here or maybe `actuallyReport` has a different meaning than it implies?